gen/struct_test.go

```diff
@@ -877,112 +877,131 @@ func TestStructWithDefaults(t *testing.T) {
 
 func TestStructJSON(t *testing.T) {
 	tests := []struct {
-		v interface{}
-		j string
+		v, w interface{}
+		j    string
```

Comment:
`v` is obviously short for `value`. `j` is also obviously for `JSON`. What does `w` mean? It might be useful to extract the three cases that make use of the `w` field from the table test, stand-alone. Using positional fields for a {k, v} duple is pretty clean.